templates/node/lib/services/service.js.twig

```diff
@@ -64,20 +64,35 @@ class {{ service.name | caseUcfirst }} extends Service {
                 let id = undefined;
                 let response = undefined;
 
+                let counter = 0;
                 const totalCounters = Math.ceil(size / client.CHUNK_SIZE);
 
-                for (let counter = 0; counter < totalCounters; counter++) {
-                        const start = (counter * client.CHUNK_SIZE);
-                        const end = Math.min((((counter * client.CHUNK_SIZE) + client.CHUNK_SIZE) - 1), size);
-                        const headers = {
+                const headers = {
 {% for parameter in method.parameters.header %}
-                                '{{ parameter.name }}': ${{ parameter.name | caseCamel | escapeKeyword }},
+                        '{{ parameter.name }}': ${{ parameter.name | caseCamel | escapeKeyword }},
 {% endfor %}
 {% for key, header in method.headers %}
-                                '{{ key }}': '{{ header }}',
+                        '{{ key }}': '{{ header }}',
+{% endfor %}
+                };
+
+{% for parameter in method.parameters.all %}
+{% if parameter.isUploadID %}
+                if({{ parameter.name | caseCamel | escapeKeyword }} != 'unique()') {
+                        try {
+                                response = await this.client.call('get', path + '/' + {{ parameter.name }}, headers);
+                                counter = response.chunksUploaded;
+                        } catch(e) {
+                        }
+                }
```

Comment:
Indentation again. Please refer to the indentation of this block
https://github.com/appwrite/sdk-for-node/blob/master/lib/services/storage.js#L375-L379

versus what we used to have previously 
https://github.com/appwrite/sdk-for-node/blob/master/lib/services/storage.js#L357